### ISSUE TITLE
`wildcard_imports` is triggered on generated code

### ISSUE THREAD

**Issue by nazar-pc** [rust-lang/rust-clippy#17200]
### Summary

It is often completely impractical to handle explicit imports in macro-generated code, especially when submodules are generated programmatically. Anything that comes from proc macro or located under `#[automatically_derived]` should be ignored by this lint.

### Lint Name

wildcard_imports

### Version

```text
rustc 1.97.0-nightly (20de910db 2026-05-02)
binary: rustc
commit-hash: 20de910db49d3476ccf49ea79a4b22e2b5dface0
commit-date: 2026-05-02
host: x86_64-unknown-linux-gnu
release: 1.97.0-nightly
LLVM version: 22.1.4
```
